From b0300bef3fbb290605a7d9ef5b945762363fd696 Mon Sep 17 00:00:00 2001
From: David Wagner <david.wagner@intel.com>
Date: Fri, 28 Jun 2013 18:11:01 +0200
Subject: Audio/AlwaysListening: New key for the VT/SV active parameter

BZ: 120682
[CLEAN]

One of the conditions for activating the "AlwaysListening" feature is that it
has been requested by the Vtsv HAL; this is done through a setParameter().

The key is "vtsv_active" and the possible values are "true" and "false"

Change-Id: If264684c427b3c144afe5ab38d6cffc0c816cea8
Orig-Change-Id: I85898df4b4609f0d525b3edd6c7779d93d3a5555
Category: feature differentiation
Domain: AudioComms-VoiceTrigger
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: David Wagner <david.wagner@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index cbc6596..7db5415 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -107,6 +107,11 @@ __BEGIN_DECLS
 /* Screen state */
 #define AUDIO_PARAMETER_KEY_SCREEN_STATE "screen_state"
 
+/* Always Listening/VTSV */
+#define AUDIO_PARAMETER_KEY_ALWAYS_LISTENING_STATUS "vtsv_active"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ON "true"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_OFF "false"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

